docs/concepts.rst

```diff
@@ -43,11 +43,11 @@ The :ref:`recoverWith` operation allows you to handle an event stream failure.
 Stream Failure vs. Application Errors
 `````````````````````````````````````
 
-Stream failures are different from *application errors*.  A stream failure indicates that an event stream *cannot* produce more events.  Application errors may or may not indicate an event stream failure.  For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a wifi drop.  In contrast, an application error may occur in application logic that receives a WebSocket message and can't process because the message application state has changed.
+Stream failures are different from *application errors*.  A stream failure indicates that an event stream *cannot* produce more events.  Application errors may or may not indicate an event stream failure.  For example, an event stream representing messages arriving on a WebSocket *fails* if the WebSocket is disconnected unexpectedly due to a wifi drop.  In contrast, an application error may occur in application logic when it receives a WebSocket message that it can't process because the application state has changed.
```

Comment:
Add a `the` before `application logic`:

> ... may occur in the application logic ...

Similar to how you wouldn't say: "An accident may happen in car seat."

Reply:
Good catch, thanks.  The definite article makes sense since it's referring back to the specific application logic mentioned in the previous sentence, rather than referring to application logic in general.

I split and refactored this paragraph a good bit more.  Let me know what you think.